src/pages/index.page.tsx

```diff
@@ -14,7 +18,7 @@ const Home: NextPage = () => {
       </Head>
 
       <main className={styles.main}>
-        <h1 className={styles.title}>
+        <h1 className={styles.title} data-test-id={homeDataTestIds.heading}>
```

Comment:
Because of adding `configure({ testIdAttribute: 'data-test-id' });` in the jest configuration so it won't show this `data-test-id` in our development, right?

Reply:
This just lets Jest know which attribute it should use to find the element by testId. For the `data-test-id` attribute, it should show as it's just a normal attribute